doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_sdfw_sdrfw_update.rst

```diff
@@ -0,0 +1,54 @@
+.. _ug_nrf54h20_suit_sdfw_sdrfw_update:
+
+#############################
+Secure Domain Firmware Update
+#############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Background
+**********
+
+Both Secure Domain Firmware and Secure Domain Firmware Recovery are launched on a core responsible for establishing Root of Trust as a Secure Domain. Due to security requirements update process of these binaries differs significantly from regular updates of Local Domains.
+In order to ensure that a trusted execution can be established on the device, a valid Secure Domain Firmware Recovery has to be present on the device prior to performing an Secure Domain Firmware update.
+An update is started only if the update candidate can be verified by Secure Domain ROM using a key matching the requested update type.
+
+
+SUIT attempts to provide flexibility with regard to launching and updating firmware while allowing certain parts of the code base to remain unmodifiable for end users. Although there are various implementations possible, a platform-specific layer that maps logical firmware blocks to their corresponding memory ranges of a device is a straightforward approach for simple systems. The manifest(s) can describe actions needed to launch or update such firmware block.
+For certain areas, nrf54 family uses such approach, however Secure Domain Firmware and Secure Domain Firmware Recovery cannot be simply copied to destination address in order to replace the old firmware - its design relies on Secure Domain ROM launching and updating the binaries.
```

Comment:
```suggestion
The SUIT framework allows flexible firmware launching and updating while keeping parts of the codebase unmodifiable by end users.
A common approach is to use a platform-specific layer that maps firmware blocks to device memory ranges, simplifying updates for basic systems.
Manifests can describe actions required to launch or update each firmware block.
In the nRF54H20, this approach is used in some areas, but the SDFW and the recovery image of the SDFW require the Secure Domain ROM to handle launching and updates, rather than direct copying to a destination address.
```